warp10/src/main/java/io/warp10/continuum/Tokens.java

```diff
@@ -345,11 +346,11 @@ public static void enableCheckAttributes() {
    */
   public static Map<String,String> labelSelectorsFromReadToken(ReadToken rtoken) {
     
-    Map<String,String> labelSelectors = new HashMap<String,String>();
+    Map<String,String> labelSelectors = new LinkedHashMap<String,String>();
```

Comment:
In that case, I think it'd be nicer if `labelSelectorsFromReadToken` explicitly returns a `LinkedHashMap`.